source/package.json

```diff
@@ -15,11 +15,11 @@
   ],
   "stack": {
     "stackname": "DUS",
-    "region": "%%REGION%%"
+    "region": "us-west-2"
```

Comment:
Should leave in the "%%REGION%%" value here

Reply:
Fixed.